<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>329</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 65, 76)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pix_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_12">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_13">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_19">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_18">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_16">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_20">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_22">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_24">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_23">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pix_25">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 84, 152);
color: rgb(56, 65, 76)</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="digit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Droid Sans Mono</family>
      <pointsize>150</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 84, 152)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="trainNumber">
       <property name="styleSheet">
        <string notr="true">color:rgb(239, 239, 239)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rateNumber">
       <property name="styleSheet">
        <string notr="true">color:rgb(239, 239, 239)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>102</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="trainLabel">
       <property name="styleSheet">
        <string notr="true">color:rgb(239, 239, 239)</string>
       </property>
       <property name="text">
        <string>Training number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rateLabel">
       <property name="styleSheet">
        <string notr="true">color:rgb(239, 239, 239)</string>
       </property>
       <property name="text">
        <string>Learning rate:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QSlider" name="trainSlider">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="rateSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
